rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/factory/DefaultGraphFactory.java

```diff
@@ -104,37 +108,12 @@ public void setSingletonGraph(final boolean singletonGraph) {
         final Graph.Builder builder = new Graph.Builder();
         builder.storeProperties(storeProperties);
 
-        final String graphConfigPath = System.getProperty(SystemProperty.GRAPH_CONFIG_PATH);
-        if (null != graphConfigPath) {
-            builder.config(Paths.get(graphConfigPath));
-        }
+        builder.config(getConfigPath());
 
         for (final Path path : getSchemaPaths()) {
             builder.addSchema(path);
         }
 
-        final String graphId = System.getProperty(SystemProperty.GRAPH_ID);
```

Comment:
Might be worth leaving this bit in for a bit longer as it was only recently deprecated.